README.md

```diff
@@ -4,4 +4,26 @@
 
 This repository contains sources of the UAVCAN website.
 
-How to run the website locally: https://help.github.com/articles/using-jekyll-with-pages.
+## Cloning the repository
+`git clone https://github.com/UAVCAN/uavcan.github.io.git`
+## Dependencies
+- [Ruby](https://www.ruby-lang.org/en/) version 2.0.0 +  
+Download the [archive](https://cache.ruby-lang.org/pub/ruby/2.3/ruby-2.3.0.tar.gz) and unpack it.  
+Enter the derictory where you unpacked the archive and execute: `./configure && sudo make install` 
```

Comment:
I doubt this is the most optimal solution. [`make install`-way must not be used unless there is no other way](https://wiki.debian.org/DontBreakDebian#Don.27t_.27make_install.27); luckily, in our case there is another way: install the correct version of Ruby, and activate it using [RVM](https://rvm.io/).

I suggest to replace this with e.g. Ubuntu-specific instructions, or just keep it generic on the assumption that whoever cares will be able to figure out how to install Ruby on their own.